sae_lens/training/sae_trainer.py

```diff
@@ -204,15 +204,6 @@ def fit(self) -> TrainingSAE:
         pbar.close()
         return self.sae
 
-    @torch.no_grad()
-    def _estimate_norm_scaling_factor_if_needed(self) -> None:
-        if self.cfg.normalize_activations == "expected_average_only_in":
-            self.activation_store.estimated_norm_scaling_factor = (
-                self.activation_store.estimate_norm_scaling_factor()
-            )
-        else:
-            self.activation_store.estimated_norm_scaling_factor = 1.0
-
```

Comment:
moved this into `ActivationsStore` as it's setting a property on it, from a method in it. Also having it in `ActivationsStore` keeps the handling of the lifecycle of `ActivationsStore.estimated_norm_scaling_factor` together